action.yml

```diff
@@ -1,37 +1,54 @@
-    description: 'JSON payload to send to Slack if webhook route. If not supplied, json from GitHub event will be sent instead'
+  payload:
+    description: "Attributes that create the content of the request"
     required: false
-  payload-delimiter: # custom delimiter used to flatten nested values in the JSON payload
-    description: 'Custom delimiter used to flatten nested values in the JSON payload. If not supplied, defaults to a period (".").'
+  payload-delimiter:
+    description: "Field seperators for nested attributes in the payload"
     required: false
-  payload-file-path: # path to JSON payload to send to Slack via webhook
-    description: 'path to JSON payload to send to Slack if webhook route. If not supplied, json from GitHub event will be sent instead. If payload is provided, it will take preference over this field'
+  payload-file-path:
+    description: "Path to a file containing the valid JSON payload"
     required: false
-  payload-file-path-parsed:
-    description: 'Replace templated variables in the JSON payload file with values from the GitHub context and environment variables'
-    default: true
+  payload-templated:
+    default: "false"
+    description: "If templated variables in the input payloads should be replaced"
     required: false
-  update-ts: # The timestamp of a previous message posted to update it instead of posting a new message
-    description: 'The timestamp of a previous message posted. It will update the existing message instead of posting a new message'
+  proxy:
+    description: "An optional alternate proxied route to for HTTPS connections"
+    required: false
+  retries:
+    default: "5"
+    description: "The method to use when performing retried requests"
+  token:
+    description: "The authentication value used with the Slack API"
+    required: false
+  webhook:
+    description: "A location for posting request payloads"
+    required: false
+  webhook-type:
+    description: "Posting with either an incoming webhook or webhook trigger"
     required: false
 outputs:
-  time: # id of output
-    description: 'The time'
-  thread_ts: # threaded timestamp on the message that was posted when using bot token
-    description: 'The timestamp on the message that was posted into Slack when using bot token'
-  ts: # timestamp of message posted
-    description: 'The timestamp on the message that was posted into Slack when using bot token'
-  channel_id: # Id of the channel. If channel name was used as input we can get ID from output for later use when updating message or posting to thread.
-    description: 'The channel id of the message that was posted into Slack when using bot token'
+  time:
+    description: "The Unix epoch time that the job completed"
+  ok:
+    description: "If the requested completed without errors"
+  channel_id:
+    description: "The channel ID of a message posted using a token"
+  response:
+    description: "A stringified version of the Slack API response"
```

Comment:
Correct! And good callout. It's not so clear from this description, but this `response` exists for **both** techniques that use a `webhook` or `method`. It now reads as this:

https://github.com/slackapi/slack-github-action/blob/bb5ce26ae1d51fd224023aaff35cfaf6c05289bf/action.yml#L43-L44

I find the current wording somewhat unclear since it doesn't specify "both method and webhook techniques" but also don't believe the `response` from `webhook` techniques is super useful...

These [passing](https://github.com/slackapi/slack-github-action/blob/bb5ce26ae1d51fd224023aaff35cfaf6c05289bf/test/webhook.spec.js#L14-L70) and [failing](https://github.com/slackapi/slack-github-action/blob/bb5ce26ae1d51fd224023aaff35cfaf6c05289bf/test/webhook.spec.js#L92-L148) tests match actual `webhook` responses which makes me think the `ok` response attribute - not `response` - will be most useful for these techniques. However, I still think including `response` makes sense for ✨ completeness ✨